.gitignore

```diff
@@ -114,3 +114,8 @@ data/catch-ignore.yml
 release_config.json
 web/userdata.js
 location.json
+
+#Multiple config
+configs/
+!configs/config.json.example
+!configs/release_config.json.example
```

Comment:
I think once you have both your files in the configs folder I don't see why we couldn't merge